perception/euclidean_cluster/src/voxel_grid_based_euclidean_cluster_node.cpp

```diff
@@ -61,10 +61,7 @@ void VoxelGridBasedEuclideanClusterNode::onPointCloud(
   cluster_pub_->publish(output);
 
   // build debug msg
-  if (debug_pub_->get_subscription_count() < 1) {
-    return;
-  }
-  {
+  if (debug_pub_->get_subscription_count() >= 1) {
```

Comment:
```suggestion
  if (1 <= debug_pub_->get_subscription_count()) {
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2108/commits/1246d3b592bdd66f8bbe9718454f01356a115062